src/stores/ui.ts

```diff
@@ -0,0 +1,23 @@
+/** Manages bcros account data */
+export const useBcrosDashboardUi = defineStore('bcros/dashboardUi', () => {
+  const uiIsLoading = ref([])
+
+  const trackUiLoadingStart = (trackingUuid: string) => {
+    uiIsLoading.value.push(trackingUuid)
+  }
+
+  const trackUiLoadingStop = (trackerUuidDone: string) => {
+    const index = uiIsLoading.value.find(tracker => tracker === trackerUuidDone)
+    if (index !== -1) {
+      uiIsLoading.value.splice(index, 1)
+    }
+  }
+
```

Comment:
I think we should use `findIndex` since `find` will return the value not the index.

Reply:
Thank you, this is great spot